src/main/java/hudson/plugins/promoted_builds/PromotionProcess.java

```diff
@@ -73,12 +77,16 @@
      * and ${rootURL}/plugin/promoted-builds/icons/32x32/, e.g. <code>"star-gold"</code>.
      */
     public String icon;
-    
+
     /**
      * The label that promotion process can be run on.
      */
     public String assignedLabel;
-    
+    /**
+     * Tells if this promotion should be hidden.
+     */
+    public String isVisible;
```

Comment:
Not sure what you mean by this. Internally isVisible uses are correctly accounting for possibility of the value being null (unless i am missing something). It is exposed via getIsVisible() to the outside. Should i mark it somehow that it can be null?